File: test/L63.m
function [xdot] = L63(~,x)
sigma = 10;
rho = 28;
beta = 8/3;
xdot = [sigma*x(2,:) - sigma*x(1,:);...
        x(1,:)*rho - x(1,:).*x(3,:)-x(2,:);...
        x(1,:).*x(2,:) - beta*x(3,:)];
end   

File: L63.py
import numpy as np

def L63(x, t):
    # Evaluates the right hand side of the Lorenz '93 system
    sigma = 10
    rho = 28
    beta = 8/3
    xdot = [sigma*x[1] - sigma*x[0],
            x[0]*rho - x[0]*x[2] - x[1],
            x[0]*x[1] - beta*x[2]]
    return xdot